_pages/dmr.md

```diff
@@ -28,7 +28,7 @@ plug/config file to get started with and you will need your own radio id to make
 
 ## Retevis RT3s Club Plug
 
-**Curent File:** [BARC_ver1.6.0.rdt](/assets/dmr/BARC_ver1.6.0.rdt)
+**Curent File:** [BARC_ver1.6.0.rdt](/assets/dmr/BARC_ver1.7.0.rdt)
```

Comment:
Can you update the displayed version number to match the file name, and also change the updated date please?